app/src/main/java/org/sopt/pingle/presentation/ui/main/mypingle/MyPingleViewHolder.kt

```diff
@@ -0,0 +1,98 @@
+package org.sopt.pingle.presentation.ui.main.mypingle
+
+import android.content.Context
+import android.view.View
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import androidx.core.view.ViewCompat
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.pingle.R
+import org.sopt.pingle.databinding.ItemMyPingleBinding
+import org.sopt.pingle.domain.model.MyPingleEntity
+import org.sopt.pingle.presentation.type.CategoryType
+import org.sopt.pingle.util.context.colorOf
+import org.sopt.pingle.util.convertToCalenderDetail
+
+class MyPingleViewHolder(
+    private val binding: ItemMyPingleBinding,
+    private val context: Context,
+    private val navigateToMapList: () -> Unit,
+    private val navigateToWebViewWithChatLink: (String) -> Unit,
+    private val showDeleteModalDialogFragment: (MyPingleEntity) -> Unit,
+    private val viewClickListener: (ConstraintLayout) -> Unit
+) : RecyclerView.ViewHolder(binding.root) {
+    fun onBind(item: MyPingleEntity) {
+        with(binding) {
+            myPingleItem = item
+            badgeMyPingleCategory.setBadgeCategoryType(CategoryType.fromString(item.category))
+            tvMyPingleName.setTextColor(
+                context.colorOf(CategoryType.fromString(item.category).textColor)
+            )
+            tvMyPingleCalenderDetail.text =
+                convertToCalenderDetail(
+                    date = item.date,
+                    startAt = item.startAt,
+                    endAt = item.endAt
+                )
+            tvMyPingleCalenderRecruitmentDetail.text = context.getString(
+                R.string.my_pingle_participant,
+                item.curParticipants,
+                item.maxParticipants
+            )
+
+            if (item.isOwner) {
+                ivMyPingleOwner.visibility = View.VISIBLE
```

Comment:
```suggestion
          ivMyPingleOwner.visibility = if (item.isOwner) View.VISIBLE else View.INVISIBLE
```
이런 식으로 써주세용

그리고 tv랑 iv도 Owner일 때 아닐 때 전부 분기처리 해줘야 할 것 같습니다.